### DISCUSSION TITLE
Ability to create pr from mobile app

### DISCUSSION THREAD

**Discussion by bannsec** [community/community#144092] [Mobile]
There really needs to be a way to create a pr from the mobile app. For instance, I have a branch already ready, and I just want to put in a pr from that branch. I should be able to do this from the mobile app. It's currently impossible.

**Comment by kangarruu** [comment]
Thanks for writing in, I filed this request with our team so we could get it on our roadmap. Be sure to keep an eye on our release notes for updates!

**Comment by aconsuegra** [comment]
Hi everyone! We have added the possibility to create pull requests from existing branches on GitHub Mobile. It's available in version 1.193 and you can read more about the announcement in https://github.blog/changelog/2025-01-22-create-pull-request-from-an-existing-branch-on-github-mobile/ 

Let us know if you have any feedback!
